src/plugins/lightbox/lightbox.js

```diff
@@ -21,6 +21,11 @@ var componentName = "wb-lbx",
 	modalHideSelector = "#wb-tphp, body > header, body > main, body > footer",
 	$document = wb.doc,
 	callbacks, i18n, i18nText,
+	defaults = {
+
+		// exclude 'times' from screen reader with aria-hidden span
+		closeMarkup: "<button type='button' class='mfp-close' title='%title%'><span class='mfp-close' aria-hidden='true'>&times;</span></button>"
```

Comment:
This markup for that button should be revised. Title element should be avoided, use `.wb-inv` as a design pattern like how we do for other similar feature.

Reply:
Let improve it considering you are currently revising that code. Currently that button fallback on the very last way to provide a label according to the Accessible name and description computation algorithm - https://www.w3.org/TR/accname-1.2/#mapping_additional_nd_name

I propose the following which will do similar as you suggested and still leverage `wb-inv` CSS class:
```suggestion
		closeMarkup: "<button type='button' class='mfp-close'><span class='mfp-close' aria-hidden='true'>&times;</span><span class='wb-inv'>%title%</span></button>"
```